support/azure/azure-storage/files-troubleshoot-smb-authentication.md

```diff
@@ -390,6 +390,20 @@ The solution is to add the privateLink FQDN to the storage account's Azure AD ap
 1. Update any internal DNS references to point to the private link.
 1. Retry mounting the share.
 
+Error AADSTS50105- 
+
+The request was interrupted by the following challenge: AADSTS50105: Your administrator has configured the application "Enterprise application name" to block users unless they are specifically granted ('assigned') access to the application.  The signed in user '{EmailHidden}' is blocked because they are not a direct member of a group with access, nor had access directly assigned by an administrator. Please contact your administrator to assign access to this application."
```

Comment:
```suggestion
The request was interrupted by the following challenge: AADSTS50105: Your administrator has configured the application "Enterprise application name" to block users unless they are specifically granted ('assigned') access to the application.  The signed in user '{EmailHidden}' is blocked because they are not a direct member of a group with access, nor had access directly assigned by an administrator. Please contact your administrator to assign access to this application.
```